asset/css/variables.less

```diff
@@ -100,6 +103,21 @@
 
 @card-border-color: @base-gray-light;
 
+@schedule-element-fields-bg: @default-input-bg;
+@schedule-element-default-text-color: @default-bg;
+@schedule-element-text-color: @base-primary-color;
+
+@schedule-element-fields-selected-bg: @base-primary-color;
+@schedule-element-fields-hover-bg: fade(@base-primary-dark, 30);
+
+@schedule-element-fields-selected-hover-bg: #00A5CF;
```

Comment:
This is still initialized with an absolute value.

Reply:
✅